internal/integration_test/spectest/spectest.go

```diff
@@ -836,38 +836,3 @@ func requireStripCustomSections(t *testing.T, binary []byte) []byte {
 	}
 	return out.Bytes()
 }
-
-// TestBinaryEncoder ensures that binary.EncodeModule produces exactly the same binaries
-// for wasm.Module via binary.DecodeModule modulo custom sections for all the valid binaries in spectests.
-func TestBinaryEncoder(t *testing.T, testDataFS embed.FS, enabledFeatures api.CoreFeatures) {
```

Comment:
deleted this test which is used to develop our own fuzzer, but now it's gone. so I deleted this